contents/tutorials/test-frontend-feature-flags.md

```diff
@@ -70,14 +118,14 @@ posthog.init(
   }
 );
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <PostHogProvider client={posthog}>
-      <App />
-    </PostHogProvider>
-  </React.StrictMode>
-);
+const root = document.getElementById('root');
+createRoot(root).render(
+  <StrictMode>
+  	<PostHogProvider client={posthog}>
+	  <App />
+  	</PostHogProvider>
+  </StrictMode>,
+)
 ```
 
 With this setup, events are automatically captured, and we can set up our [React feature flag](/tutorials/react-feature-flags).
```

Comment:
The below section needs a slight week because the file is `App.jsx`

Reply:
Wdym with "slight week"?